tla-io/src/main/scala/at/forsyte/apalache/io/quint/Quint.scala

```diff
@@ -416,7 +416,7 @@ class Quint(quintOutput: QuintOutput) {
 
     // the quint builtin operator representing match expressions looks like
     //
-    // matchVariant(expr, "F1", elim_1, ..., "Fn", elim_n)
+    // matchVariant(expr, "F1", elim_1, ..., "Fn", elim_n, "_", defaultElim)
```

Comment:
Perhaps a notation like this would help indicate that this is optional
```suggestion
    // matchVariant(expr, "F1", elim_1, ..., "Fn", elim_n[, "_", defaultElim])
```